src/style.css

```diff
@@ -1 +1,99 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --primary-color: #000;
+  --accent-color: #00acdc;
+  --background-color: #fff;
+  --header-line-height: 60px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
![image](https://github.com/mate-academy/layout_moyo-header/assets/59209337/4b2b80f4-561e-48f8-8831-77b187bb6ab2)

Reply:
Hi! 👋  As a "fallback" system, it shares a common generic-family as "sans-serif", but added an additional family-name of 'Helvetica' . Done :white_check_mark: